src/fragments/lib/auth/js/advanced.mdx

```diff
@@ -169,72 +169,59 @@ const App = () => {
 
 ### Google sign-in (React)
 
+
 ```js
 import React, { useEffect } from 'react';
+import jwt from 'jwt-decode'
 import { Auth } from 'aws-amplify';
-// To federated sign in from Google
-const SignInWithGoogle = () => {
 
+const SignInWithGoogle = () => {
     useEffect(() => {
-        const ga = window.gapi && window.gapi.auth2 ? 
-            window.gapi.auth2.getAuthInstance() : 
-            null;
-
-        if (!ga) createScript();
-    }, [])
-
-    const signIn = () => {
-        const ga = window.gapi.auth2.getAuthInstance();
-        ga.signIn().then(
-            googleUser => {
-                getAWSCredentials(googleUser);
-            },
-            error => {
-                console.log(error);
-            }
-        );
-    }
-
-    const getAWSCredentials = async (googleUser) => {
-        const { id_token, expires_at } = googleUser.getAuthResponse();
-        const profile = googleUser.getBasicProfile();
-        let user = {
-            email: profile.getEmail(),
-            name: profile.getName()
-        };
-        
-        const credentials = await Auth.federatedSignIn(
-            'google',
-            { token: id_token, expires_at },
-            user
-        );
-        console.log('credentials', credentials);
-    }
+        /// Check for previous Google client initialization
```

Comment:
Nit: `///`, also maybe say "Check for an existing Google client initialization" instead